packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -373,7 +375,7 @@ export class UnfurlService {
         const buffer = await this.saveScreenshot(
             imageId,
             cookie,
-            url,
+            url.replace('localhost:8080', 'localhots:3000'), // This is needed for local testing, it will do nothing on prod
```

Comment:
there is a typo here `localhots` <-

Also this is only needed if you don't do `yarn build` in dev (since that serves the latest files on `localhost:8080` just like in prod). This could confuse somebody in future who is actually running this on `localhost:8080` locally

Reply:
ok, I'll remove it then